tasks/82_bouncing-balls/bouncing-balls.js

```diff
@@ -1,15 +1,18 @@
 export const bouncingBall = (height, bounce, window) => {
```

Comment:
it's ok for now, but the `window` variable exists in browser environment, so please avoid using this name for variable